cmd/typed-errors.go

```diff
@@ -58,3 +58,6 @@ var errInvalidRange = errors.New("Invalid range")
 // errInvalidRangeSource - returned when given range value exceeds
 // the source object size.
 var errInvalidRangeSource = errors.New("Range specified exceeds source object size")
+
+// error returned by disks which are to be initialized.
+var errNotFirstDisk = errors.New("Wait for first server to format the disks")
```

Comment:
better do:
var errNotFirstDisk = errors.New("not first disk")

and at the other place do:
```
case errNotFirstDisk:
   console.Println("Wait for first server to format the disks")
   continue
```

else do:
var errFormatWait = errors.New("Wait for first server to format the disks")